archive.html

```diff
@@ -0,0 +1,17 @@
+---
+title: Archive
+---
+
+{% for category in site.categories %}
+  <h3>{{ category[0] }}</h3>
+  <style type="text/css">
+    ul {
+      padding-left: 2rem;
+    }
+  </style>
```

Comment:
Quick and dirty css injection - not worried about it because I don't expect it to be a pattern. Unordered lists just look awful in md.